R/wb_styles.R

```diff
@@ -734,7 +734,7 @@ get_cell_style <- function(wb, sheet, cell) {
 #'
 #' \dontrun{
 #' # look at the beauty you've created
-#' wb_open(wb)
+#' if (interactive()) wb_open(wb)
```

Comment:
This looks weird. It is already in a dontrun section and wb_open should bring the interactive check?

Reply:
I've seen this added into some packages for their examples.  Maybe just be another sort of safeguard to make sure files aren't popping up.  I was checking some of the examples with `devtools::run_examples()` and some where still opening things.

Reply:
ok